### ISSUE TITLE
Upgrading to Pixi 4?

### ISSUE THREAD

**Issue by PulsarMedia** [kittykatattack/hexi#54]
Hi,

Is it possible to update the pixi engine to version 4? I am running into texture issues on mobile devices and after digging around the Pixi forums i've come to learn that most of the texture issues i am experiencing with version 3 have been fixed in version 4.

If its easy to do can you provide me with some instructions on how to go about updating the engine?

Thanks.

**Comment by kittykatattack** [comment]
@PulsarMedia Instead of using `hexi.min.js`, you could try linking these three files, in order: `pixi.min.js` (the latest version 4), `core.js`, and `modules.js`.
This did work with earlier versions of Pixi v4, but I'm not sure if current versions have introduced breaking API changes that prevent v3 compatibility.
Upgrading to v4 is something I started a few months ago, but it proved to be a major task and is unfortunately not something I'm able to do soon (see: https://github.com/kittykatattack/hexi/issues/32.)
Have you asked on the Pixi forums or Pixi's Github issues whether there's s fix for the texture rendering problems you're encountering?

**Comment by PulsarMedia** [comment]
I was able to figure out what the texture issue was yesterday - i was loading an animation spritesheet with a width of around 5000px and the texture was rendering black on mobile devices. I shortened the width to 3900px and that fixed the issue.

If i run into any other texture issues down the road i will bring it to your attention. I would love to see an update to the Hexi engine running on Pixi 4 though...after reading up on Pixi 4 it looks like they have made significant improvements to the engine. I know its a lot of work to update it but it would be beneficial for Hexi users.

Overall i'm really enjoying the Hexi engine 👍
